sycl/doc/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+if (DOXYGEN_FOUND)
+if (LLVM_ENABLE_DOXYGEN)
+  set(abs_srcdir ${CMAKE_CURRENT_SOURCE_DIR})
+  set(abs_builddir ${CMAKE_CURRENT_BINARY_DIR})
+
+  if (HAVE_DOT)
+    set(DOT ${LLVM_PATH_DOT})
+  endif()
+
+  # TODO add support for external search
```

Comment:
External search of what exactly? (Just fighting my personal incompetence)

Reply:
@AGindinson I need to reword this. Doxygen has support for external search engines. LLVM projects seem to make use of it. But we currently have no need for an external search engine.